client/web/src/insights/components/visibility-picker/VisibilityPicker.tsx

```diff
@@ -67,12 +67,17 @@ export const VisibilityPicker: React.FunctionComponent<VisibilityPickerProps> =
         }
     }
 
+    const possibleOrg = organizations.find(org => org.id === value)
+
+    const descriptionText = !possibleOrg?.displayName
+        ? 'This insight will be visible only on your personal dashboard. It will not be show to other users in your organization.'
+        : `This insight will be visible to all users in the ${possibleOrg.displayName} organization who have enabled code insights.`
```

Comment:
- Fix typo: show -> shown
- Minor sentence reorder: "only" before the verb ("be")
- Semantic change: Change "on your personal dashboard" to "you" because it will also be shown on directory pages (I think @Joelkw pointed this out). If that is wrong please ignore this suggestion.
- If `displayName` is not set, use the `name`.
```suggestion
        ? 'This insight will only be visible to you. It will not be shown to other users in your organization.'
        : `This insight will be visible to all users in the ${possibleOrg.displayName || possibleOrg.name} organization who have enabled code insights.`
```